src/answers-utilities.ts

```diff
@@ -14,9 +14,20 @@ export default {
   searchThroughFacet(facet: DisplayableFacet, searchTerm: string): DisplayableFacet {
     return {
       ...facet,
-      options: facet.options.filter(o =>
-        isLevenshteinMatch(o.displayName.toLowerCase(), searchTerm.toLowerCase())
-      )
+      options: facet.options.filter(o => isCloseMatch(o.displayName, searchTerm))
     };
-  }
-};
\ No newline at end of file
+  },
+  isCloseMatch
```

Comment:
Does this export work properly? There can only be one `default` export per module. It looks as if we're attempting to have two default exports. If this module is to export both `searchThroughFacet` and `isCloseMatch`, one should either be the `default` or we should have two named exports.

Reply:
this is exporting an single object that has two functions. I think I would just prefer 2 named exports, and an `import * as AnswersUtilities from './answers-utilities` wherever this gets imported if we really want to group things

Reply:
It's not giving any errors with the export like this and I'm able to use it in the starter repo locally. But, I can switch to having two named exports instead. That would also remove the forgotten exports warning in the API report.